alchemy/src/apply.ts

```diff
@@ -162,33 +165,83 @@ async function _apply<Out extends Resource>(
       props: state.oldProps,
       state,
       replace: () => {
+        if (phase === "create") {
+          throw new Error(
+            `Resource ${resource[ResourceKind]} ${resource[ResourceFQN]} cannot be replaced in create phase.`,
+          );
```

Comment:
"Replace cannot be triggered during the create phase

Reply:
this would omit what resource is causing the problem